c++/src/capnp/rpc.c++

```diff
@@ -3142,13 +3142,16 @@ public:
     traceEncoder = kj::mv(func);
   }
 
+  kj::Promise<void> getAcceptLoopPromise() { return acceptLoopPromise.addBranch(); }
+
 private:
   VatNetworkBase& network;
   kj::Maybe<Capability::Client> bootstrapInterface;
   BootstrapFactoryBase& bootstrapFactory;
   kj::Maybe<SturdyRefRestorerBase&> restorer;
   size_t flowLimit = kj::maxValue;
   kj::Maybe<kj::Function<kj::String(const kj::Exception&)>> traceEncoder;
+  kj::ForkedPromise<void> acceptLoopPromise = nullptr;
```

Comment:
Instead of a forked promise, how about we tell people that they can only call `getAcceptLoopPromise()` once? That's something I commonly do in KJ code in order to avoid unnecessary promise forking.

Reply:
Ahh, makes sense.